webrender/src/gpu_cache.rs

```diff
@@ -468,9 +468,8 @@ impl<'a> GpuDataRequest<'a> {
         self.texture.pending_blocks.extend_from_slice(blocks);
     }
 
-    /// Consume the request and return the number of blocks written
-    pub fn close(self) -> usize {
```

Comment:
I'd prefer to keep this as ```close``` taking ```self``` as a parameter, that prevents it from being used again after the close is called.